### DISCUSSION TITLE
Question regarding initial dataset

### DISCUSSION THREAD

**Discussion by roger13231** [deepmodeling/dpgen#1575] [Q&A] [answered]
Hi everyone, I am using dpgen to train a potential for high entropy alloy, and I have a question regarding the accuracy of AIMD and dpgen fp.

Eariler, I was using deepmd-kit seperately, and prepared around 200k AIMD data with only gamma point at temperature from 300 K to 3300 K. Now, I want to use dpgen to explore more configurations. According to my 128 HEA structure, only gamma point have a kpsacing of arount 0.53, and my convergence test for kspacing is around 0.20 (333 KPOINTS) to reach 1meV/atom. In this way, I need to use 0.20 for my 02.fp in dpgen.

My question is, can I still use my AIMD data in initial dataset (e.g. take 1 frame every 100 frame, so 2000 AIMD frame)? Will the low accuracy (111 kpoint) have conflict with new fp data?
Or, I should abandon my AIMD data. Maybe only use a few 300 K data, and constructure my dataset with dpgen fp.

**Comment by wanghan-iapcm** [comment, marked as answer]
Mixing low-accuracy data with high-accuracy data would reduce the model's overall accuracy.

It is recommended to relabel some of the AIMD frames using high-standard DFT calculations.

**Reply by roger13231** [reply]
Get it. Thank you, Han.
